test/query.test.js

```diff
@@ -19,4 +19,27 @@ describe('Testing query route', () => {
     expect(searchTerm.length).toBeGreaterThan(256);
     expect(res.status).toBe(400);
   });
+
+  test('Testing with empty text', async () => {
+    const res = await request(app).get('/query?filter=post');
+    expect(res.status).toBe(400);
+  });
+
+  test('Testing with empty filter', async () => {
+    const searchTerm = encodeURIComponent('I Love Telescope');
+    const res = await request(app).get(`/query?text=${searchTerm}`);
+    expect(res.status).toBe(400);
+  });
+
+  test('Testing with invalid filter', async () => {
+    const searchTerm = encodeURIComponent('I Love Telescope');
+    const res = await request(app).get(`/query?text=${searchTerm}&filter=test`);
+    expect(res.status).toBe(400);
+  });
+
+  test('Testing with valid filter', async () => {
```

Comment:
Let's do another test like this, but for `author` so we have both.

Reply:
Incorporated in [7dd761b](https://github.com/Seneca-CDOT/telescope/pull/1334/commits/7dd761bc7b9281cd31a572a30424e0c41235cb6b)